ui/pages/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -102,9 +96,19 @@ const mapStateToProps = (state, ownProps) => {
   const { name: fromName } = identities[fromAddress];
   const toAddress = propsToAddress || txParamsToAddress;
 
+  const tokenList = getTokenList(state);
+  const useStaticTokenList = getUseStaticTokenList(state);
```

Comment:
Is the casing of the static and dynamic lists returned from the controller different?

Reply:
The addresses in `contract-metadata` are checksum addresses and the addresses we get from the API are all lower case.